### DISCUSSION TITLE
Ready to develop the code

### DISCUSSION THREAD

**Discussion by AnjaliSheshappa** [AnjaliSheshappa/Tafe22S104#2] [General]
Hi guys! hope you all doing well!

Have you guys started coding

**Comment by thuc-lam** [comment]
Hi Anjali!

I've just started my coding, hope you guys are off to a good start as well.

**Comment by AnjaliSheshappa** [comment]
Hi Guys! 

I have finished my coding, yet to push it to the remote repository! hopefully, by tonight I will push the code.

**Comment by Matt0244** [comment]
Hi,

I have done my C# code as well

**Comment by AnjaliSheshappa** [comment]
Hi, Guys 

shall we merge the code?
